low-level/framework/base/sspl_constants.py

```diff
@@ -57,18 +57,52 @@
 CONSUL_HOST = consulhost
 CONSUL_PORT = consulport
 SSPL_SETTINGS = {
-        "ACTUATORS" : ["Service", "RAIDactuator", "Smartctl", "NodeHWactuator"],
-        "CORE_PROCESSORS" : ("RabbitMQegressProcessor", "RabbitMQingressProcessor", "LoggingProcessor"),
-        "DEGRADED_STATE_MODULES" : ("ServiceWatchdog", "RAIDsensor", "NodeData", "IEMSensor", "NodeHWsensor",
-                            "DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                            "NodeControllerMsgHandler", "SASPortSensor", "MemFaultSensor", "CPUFaultSensor"),
-        "MESSAGE_HANDLERS" : ("DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                        "NodeControllerMsgHandler"),
-        "SENSORS" : ["ServiceWatchdog", "RAIDsensor", "NodeData",
-            "IEMSensor", "NodeHWsensor",
-            "SASPortSensor", "MemFaultSensor", "CPUFaultSensor", "RAIDIntegritySensor"]
+        "REALSTORSENSORS": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["RealStorFanSensor", "RealStorPSUSensor",
+                "RealStorControllerSensor", "RealStorDiskSensor", "RealStorSideplaneExpanderSensor",
+                "RealStorLogicalVolumeSensor", "RealStorEnclosureSensor",],
+        },
+        "NODEHWSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [ "NodeHWsensor",],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": [ "NodeHWsensor",],
+        },
+        "SYSTEMDWATCHDOG": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": [],
+        },
+        "RAIDSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": ["RAIDsensor", ],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["RAIDsensor", "RAIDIntegritySensor"],
+        },
+        "CATCHALL": {
```

Comment:
@pranav-seagate Will we disable e.g. SASportsensor, Mem/CPU fault sensors, etc. too for the virtual server?

Reply:
At a quick glance these appear to be the normal threaded sensors, so it should be simple to add them to the list of configurable sensors by putting the 'should_monitor' key in the appropriate conf file section.

Reply:
We need to disable e.g. SASportsensor, Mem/CPU fault sensors, etc. too for the virtual server case, please make necessary changes if it is not done currently.

Reply:
Pushed.